configs/peer.template.toml

```diff
@@ -0,0 +1,75 @@
+## For the full reference, go to (TODO put link)
+
+## You can use another TOML file to extend from.
+## For a single file extension:
+# extends = "./base.toml"
+## Or, for a chain of extensions:
+# extends = ["base-1.toml", "base-2.toml"]
+
+# chain_id =
+# public_key =
+# private_key.digest_function =
+# private_key.payload =
+
+[network]
+# address =
+# block_gossip_period = "10s"
+# max_blocks_per_gossip = 4
+# max_transactions_per_gossip = 500
+# transaction_gossip_period = "1s"
```

Comment:
This naming is confusing for me. Any ideas to improve?

Maybe:

- `block_gossip_size`, `block_gossip_period`, `transaction_gossip_size`, `transaction_gossip_period`
- or maybe `_amount`, or `_max_amount`
- it used to be `batch_size`, maybe restore this idk